courses/developingapps/python/datastore/start/quiz/webapp/routes.py

```diff
@@ -19,14 +19,14 @@
 from flask import send_from_directory
 from flask import request, redirect
 
-import questions
+from . import questions
```

Comment:
Is it possible to do this with a relative import (without the .)?

Reply:
Without relative import we are getting `ModuleNotFoundError: No module named`  error for `api` and `questions` imports. So we resolve it using explicit relative import in the Python script.

However, we can also use absolute import as below.

```
from quiz.api import api
from quiz.webapp import questions
```

Please, let me know your thoughts on this.

Reply:
let's use the absolute import referenced in you comment - thanks Sweta!

Reply:
@mylenebiddle updated the python files with the absolute import.
- Thanks!